cmake/Corrosion.cmake

```diff
@@ -765,6 +765,9 @@ function(_add_cargo_build out_cargo_build_out_dir)
         list(APPEND corrosion_cc_rs_flags "AR_${_CORROSION_RUST_CARGO_TARGET_UNDERSCORE}=${CMAKE_AR}")
     endif()
 
+    # TODO: How to get a working host compiler properly? configure a dummy cmake project for the host?
```

Comment:
I like the idea to use a dummy cmake project for detecting the host compiler! I guess this dummy project would be generated via Make instead of XCode to not fall into the same trap of using the wrong `cc`?

Reply:
If you don't configure CMake to compile for iOS, I'm pretty sure XCode wouldn't setup the PATH with a `cc` for iOS, but provide one to compile for macos.